docs/faq/faq-felizandfable.md

```diff
@@ -0,0 +1,73 @@
+In a nutshell Fable.React and Feliz are two F# libraries which perform a similar function:
+
+* Provide a typesafe F# wrapper to allow you to interact with React.
+* Provide a way to create your own wrappers around existing React components.
+* Provide a DSL for you to consume and create React wrappers in a consistent way.
+
+## DSL differences
+The main distinction between the two libraries is that Fable.React follows a layout as follows:
+
+```fsharp
+element [ prop; prop ] [ childElement; childElement ]
+```
+
+For example:
+
+```fsharp
+h1 [ Style [ Color "Tomato" ] ] [
+    p [] [ text [ Value "Hello" ] [] ] // no props
+    p [] [ text [ Value "Another paragraph" ] [] ] // no props
+    h2 [ Style [ Color "Blue" ] ] [] // no child elements
+]
```

Comment:
This is not valid Fable.React code well it will compile but this not valid HTML.

You are setting the `Value` attributes of a `text` element instead of displaying text.

```fs
    h1 [ Style [ Color "Tomato" ] ] [
        p [] [ str "Hello" ] // no props
        p [] [ str "Another paragraph" ] // no props
        h2 [ Style [ Color "Blue" ] ] [] // no child elements
    ]
```